src/view/transaction-detail-mobile.js

```diff
@@ -40,12 +40,13 @@ const styles = StyleSheet.create({
 });
 
 const TransactionDetailView = ({ store, nav }) => (
-  <Background
+  <SplitBackground
     image={
       store.selectedTransaction.type === 'bitcoin'
         ? 'orange-gradient-bg'
         : 'purple-gradient-bg'
     }
+    bottom={color.whiteBg}
```

Comment:
To me this line doesn't do anything... So the problem this PR solves is only on the iphone XS?

Reply:
> To me this line doesn't do anything... To clarify, the problem this PR solves is only on the iphone XS?

Oh sorry. Forgot to link the issue #846. This fixes the orange bar below the SafeAreaView.